include/CppUTestExt/MockFailure.h

```diff
@@ -48,6 +48,7 @@ class MockFailureReporter
     virtual UtestShell* getTestToFail();
 
     virtual void crashOnFailure() {crashOnFailure_ = true; }
+    virtual void clear() {crashOnFailure_ = false; }
```

Comment:
I don't think I'd like crashOnFailure_ to be false in clear. This is something that I would even put in the main so that I can easily turn it on/off. It might be good to make it a command line parameter, but definitively not something you want to turn off every clear.